tool/evaluate-file.ts

```diff
@@ -84,6 +103,7 @@ export function initArgparse(subparsers : argparse.SubParser) {
     parser.add_argument('--csv', {
         action: 'store_true',
         help: 'Output a single CSV line',
+        default: true
```

Comment:
You can't default this one to true, because there is no way to turn it off. Other commands also have it default to false.

Reply:
> You can't default this one to true, because there is no way to turn it off. Other commands also have it default to false.

oh, yeah, I will revert this. `tokenized` should be fine since there is an negate option, right?

Reply:
Yes. That's basically the only reason to have a negate option...